src/libcollections/btree/node.rs

```diff
@@ -885,6 +1018,9 @@ impl<'a, K, V, NodeType> Handle<NodeRef<marker::Mut<'a>, K, V, NodeType>, marker
 }
 
 impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Leaf>, marker::KV> {
+    /// Split the underlying node into three parts. The node is truncated in place to what is
+    /// to the left of this handle, and the key and value pointed at by this handle and the
+    /// rest of the node to the right of this handle are returned separately.
```

Comment:
This is a nasty list of things, perhaps a better (less compact?) phrasing exists?